src/smc-webapp/syncdoc.coffee

```diff
@@ -79,42 +83,43 @@ class SynchronizedDocument extends AbstractSynchronizedDoc
         chat = @element.find(".salvus-editor-codemirror-chat")
         input = chat.find(".salvus-editor-codemirror-chat-input")
 
-        # send chat message
-        input.keydown (evt) =>
-            if evt.which == 13 # enter
-                content = $.trim(input.val())
-                if content != ""
-                    input.val("")
-                    @write_chat_message
-                        event_type : "chat"
-                        payload    : {content : content}
-                return false
+        # # send chat message
+        # input.keydown (evt) =>
+        #     if evt.which == 13 # enter
+        #         content = $.trim(input.val())
+        #         if content != ""
+        #             input.val("")
+        #             @write_chat_message
+        #                 event_type : "chat"
+        #                 payload    : {content : content}
+        #         return false
 
-        @chat_session.on('sync', (=>@render_chat_log()))
+        #@chat_session.on('sync', (=>@render_chat_log()))
 
-        @render_chat_log()  # first time
+        #@render_chat_log()  # first time
         @init_chat_toggle()
         @init_video_toggle()
-        @new_chat_indicator(false)
 
 
     # This is an interface that allows messages to be passed between connected
     # clients via a log in the filesystem. These messages are handled on all
     # clients through the render_chat_log() method, which listens to the 'sync'
     # event emitted by the chat_session object.
+    
+    # Writes the video chat message
     write_chat_message: (opts={}) =>
         opts = defaults opts,
             event_type : required  # "chat", "start_video", "stop_video"
             payload    : required  # event-dependent dictionary
             cb         : undefined # callback
 
         new_message = misc.to_json
-            sender_id : redux.getStore('account').get_account_id()
-            date      : new Date()
+            sender_id : "Unknown"
             event     : opts.event_type
             payload   : opts.payload
+            date      : new Date()
```

Comment:
Not sure, I think Vivek wrote that when he implemented the side chats years ago. The new Date() was there when I was working on it, I just moved it around.